gtests/test_nopython_transform.cpp

```diff
@@ -7,11 +7,22 @@
 #error "Python.h should not be included."
 #endif
 
-template <typename T>
+template <typename T, bool IsPow2>
+struct FFTTestParams
+{
+    using Type = T;
+    static constexpr bool isPow2 = IsPow2;
+};
+
+template <typename TestParam>
 class ParsevalTest : public ::testing::Test
 {
+public:
+    using T = typename TestParam::Type;
+    static constexpr bool isPow2 = TestParam::isPow2;
```

Comment:
Since the test parameter `isPow2` can be defined during compile time, so declare it as a `static constexpr`.

Reply:
Member data should use `snake_case`.